src/main/cpp/RLBotInterface/BallPrediction/inc/primitives.h

```diff
@@ -0,0 +1,101 @@
+#pragma once
+
+#include <math.h>
+#include <iostream>
+
+#include "linalg.h"
+
+struct int2{
```

Comment:
consistency of spacing between int2 and { with other structs in the code base.
And everywhere else